main.tf

```diff
@@ -123,7 +112,7 @@ data "aws_iam_policy_document" "permissions" {
     effect = "Allow"
 
     resources = [
-      "*",
+      "${var.default_role_resources}",
```

Comment:
since the default is `["*"]`, will Terraform flatten the list of lists in this particular case?

Reply:
Yes it does flatten it automatically when passed like this